### ISSUE TITLE
already existing process-map not mentionned nor featured

### ISSUE THREAD

**Issue by almereyda** [unmonastery/process-map#1]
By strolling through the links around http://unmonastery.org/bios/#process-map, I have only been sent here and to a Trello board. Unfortunately it didn't hold any card about the process-map at all.

Then I checked the other components of BIOS and found they contained completely different types of links. Therefore
1. the board is kind of the current instantiation of the process map.
2. this repository contains a website that explains these processes and visualizes them
3. the actual image file representing your approach is undocumented (which is maybe one of the only exceptions of many other possible aspects of un-words)

The third point turned out as a problem, as [we](https://publicspaceinvaders.org) have some private material to show, which I also have to get permission first.
- How did you already explore the necessary compounds of such an artifact?
- Do you have any material to share to illustrate other, though different, but similar approaches?
- Which data models and APIs do you intend to harvest for this?

**Comment by keikreutler** [comment]
Yes your conclusions are mostly correct! The process map will pull from the Trello board as a quasi-CMS - allowing other unMoaners to change the content. Each list on the process map board corresponds to a stage in unMonastery development, as mapped out by a 3-year service design framework. It'll undoubtedly evolve and change. The stages correspond to changing colors on the image, which will function as a sort of unMonastery, RPG-based board game. Lots of developing still to be done. ;)

These BIOS materials are still in "beta" - meaning their fully instantiated form will be released in the coming month, hence the lack of clear visual and descriptive information for the time being.

Currently @mozboz and I are working from this repo github.com/unmonastery/TrelloDB to extend the use of the Trello API (using Backbone and Handlebars for templating) as a CMS - James is working on the server-side caching component for the API calls, I don't believe he's pushed his progress yet. Interesting, perhaps?

As for your first and second questions, could elaborate slightly on them so that I could answer more clearly?
